Source/JavaScriptCore/dfg/DFGStrengthReductionPhase.cpp

```diff
@@ -912,16 +916,35 @@ class StrengthReductionPhase : public Phase {
                 }
                 m_graph.watchpoints().addLazily(globalObject->regExpRecompiledWatchpoint());
                 regExp = regExpObjectNode->castOperand<RegExp*>();
+            } else if (String searchString = m_node->child2()->tryGetString(m_graph); !!searchString) {
+                // String/String/String case.
+
+                size_t matchStart = string.find(searchString);
+                if (matchStart == notFound) {
+                    m_changed = true;
+                    // Preserve any checks we have.
```

Comment:
this comment isn't adding much, since that's how the code reads

Reply:
OK, dropped.